lib/src/Base/Stat/SampleImplementation.cxx

```diff
@@ -1573,6 +1573,25 @@ void SampleImplementation::sortUniqueInPlace()
   if (last + 1 < size_) erase(last + 1, size_);
 }
 
+Indices SampleImplementation::argsort() const
+{
+  Collection< std::pair<Point, UnsignedInteger> > pointsPairs(size_);
+
+  for (UnsignedInteger i = 0; i < size_; ++i)
+  {
+    Point point_i(dimension_);
+    for (UnsignedInteger j = 0; j < dimension_; ++j) point_i[j] = operator()(i, j);
+    pointsPairs[i] = std::pair<Point, UnsignedInteger>(point_i, i);
```

Comment:
Useless. You can replace the whole loop by:
```
pointsPairs[i] = std::pair<Point, UnsignedInteger>((*this)[i], i);
```
as a cont_

Reply:
Much simpler indeed. Thank you! Fixed.